assets/css/_street_view_button.scss

```diff
@@ -1,7 +1,22 @@
 .m-street-view-button {
+  /* Layout */
+  /** Prevent the button from being inlined */
+  display: block;
+  /** Prevent the button from expanding to available space */
+  width: fit-content;
+
+  /* Element Decoration */
   background-color: black;
-  color: white;
-  padding: 0.25rem;
   border-radius: 0.5rem;
+  padding: 0.25rem;
+
+  /* Text Decoration */
+  color: white;
   text-decoration: none;
+  line-height: 1;
+}
+
+/* Override `.leaflet-container a` */
+.leaflet-container .m-street-view-button {
+  color: white;
```

Comment:
Factor this out?

Reply:
We've actually already got `$white` in `_ui_kit.scss`.

Reply:
But, I believe all this is gonna be replaced by tooltip work?

Reply:
Ah, gotcha. Even if this is getting replaced soon I'd say it's worth creating a `$street-view-button-text-color` variable or something like it.

Reply:
🎉 2d56ce11c95f1bf2cb875e1f485f5c8fd2f5e432